content-src/asrouter/templates/FirstRun/FirstRun.jsx

```diff
@@ -140,10 +143,11 @@ export class FirstRun extends React.PureComponent {
 
   closeTriplets() {
     this.setState({ isTripletsContainerVisible: false });
-    // TODO: Needs to block ALL extended triplets as well
-    if (this.props.message.template === "extended_triplets") {
-      this.props.onBlock();
-    }
+
+    // Closing triplets should prevent any future extended triplets from showing up
+    setTimeout(() => {
+      this.props.onBlockById("EXTENDED_TRIPLETS_1");
+    }, TRANSITION_LENGTH);
```

Comment:
Did you try adding a "transitionend" event listener? I suppose it might be trickier to test ?